core/esmf-aspect-model-document-generators/src/main/java/org/eclipse/esmf/aspectmodel/generator/openapi/AspectModelOpenApiGenerator.java

```diff
@@ -209,9 +209,9 @@ private ObjectNode getPropertiesNode( final Optional<String> resourcePath, final
       }
 
       if ( resourcePath.isPresent() ) {
-         final List<String> dynamicParameters = Pattern.compile( "[{]\\S+[}]" ).matcher( resourcePath.get() ).results()
-               .map( match -> match.group( 0 ) ).collect( Collectors.toList() );
-         if ( !dynamicParameters.isEmpty() && !jsonProperties.isPresent() ) {
+         final List<String> dynamicParameters = Pattern.compile( "[{]\\S+?[}]" ).matcher( resourcePath.get() ).results()
+               .map( match -> match.group( 0 ) ).toList();
+         if ( !dynamicParameters.isEmpty() && jsonProperties.isEmpty()) {
```

Comment:
Please fix the code style (there should be a space before the final closing bracket), also in the other changes in the PR.